### ISSUE TITLE
v4.2.1 win Unable to open source file

### ISSUE THREAD

**Issue by Menaxerius** [bhamon/gpuPlotGenerator#55]
It seems it can't create a file, tried it with various paths and writing options, always the same error:

C:\Users\pasca\Documents\gpuPlotGenerator-4.2.1\new\Release>gpuPlotGenerator.exe generate direct C:\123456_50000_10000_2000_2
-------------------------
GPU plot generator v4.2.1
-------------------------
Author:   Cryo
Bitcoin:  138gMBhCrNkbaiTCmUhP9HLU9xwn5QKZgD
Burst:    BURST-YA29-QCEW-QXC3-BKXDL
----
Loading platforms...
Loading devices...
Loading devices configurations...
Initializing generation devices...

[ERROR] Unable to open the source file

**Comment by Menaxerius** [comment]
same error on 4.2.2

C:\Users\pasca\Documents\gpuPlotGenerator-master\new\Release>gpuPlotGenerator.exe generate direct c://plots//1223456_10000_100000_8192_2
-------------------------
GPU plot generator v4.2.2
-------------------------
Author:   Cryo
Bitcoin:  138gMBhCrNkbaiTCmUhP9HLU9xwn5QKZgD
Burst:    BURST-YA29-QCEW-QXC3-BKXDL
----
Loading platforms...
Loading devices...
Loading devices configurations...
Initializing generation devices...

[ERROR] Unable to open the source file

**Comment by Menaxerius** [comment]
Compiled using: 
CMake 3.12.0-rc1
Visual Studio 2017
Cuda 9.0 Librarys

on win 10

**Comment by bhamon** [comment]
@Menaxerius It sounds like a right problem to me.
All the mentionned paths in your previous posts are system reserved on Win10.
You can try:
- To launch your command in an elevated prompt. It is the recommended way if you want to use the "direct" writing strategy.
- To use an user owned path (like C:\Users\pasca\Documents\plots, you'll need to create the directory first).

**Comment by Menaxerius** [comment]
@bhamon 
Thanks for responding to me, The issue happend due to missing files like the Kernel files, copying them from the 4.1.1 Version helped here. On one System it worked like its supposed to, on the other (That always worked flawless for plotting) it started throwing out bad allocation with buffer with the same settings as the other one  with an "smaller" video card in it. 

I'm gonna test it on few more systems today and let you know the results

**Comment by Menaxerius** [comment]
It works on half of the computers, to make 1 plot, and throwing out an Bad allocation error if you try to make more then one plot at the same time (in one Window) and on other Computers with identical Hardware I get the same error direactly.

**Comment by bhamon** [comment]
@Menaxerius Please, if it's not done yet, use the latest stable version (4.2.2).

If you still experience bad alloc errors, ensure you have enough memory (the recap message when starting the plotter will tell you how much RAM is needed for each GPU and on the CPU side).

**Comment by Menaxerius** [comment]
Yes I am on 4.2.2. All machines have 16 GB or 32 GB RAM and 5 or more GB free and I'm only using 1 GB each and still get that error.

**Comment by bhamon** [comment]
@Menaxerius Ok then, can you provide me the following elements to help me debug this issue:
- The content of your `devices.txt` file.
- The command line you typed (you can hide your address if that's a real one).
- A screenshot of the program output.
